pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -220,7 +220,8 @@ func (k *Kubernetes) InitSvc(name string, service kobject.ServiceConfig) *api.Se
 
 // InitConfigMapForEnv initializes a ConfigMap object
 func (k *Kubernetes) InitConfigMapForEnv(name string, opt kobject.ConvertOptions, envFile string) *api.ConfigMap {
-	envs, err := GetEnvsFromFile(envFile)
+	workDir := filepath.Dir(opt.InputFiles[0])
```

Comment:
Using opt.InputFiles[0] throughout the code is kinda scary, I'd advise that we check to see that it's not an empty array.

What would be better is putting this into a function instead similar to https://github.com/kubernetes/kompose/blob/ddd5f337452952c355de5ec5d383b8beaff4c141/pkg/transformer/utils.go#L331

Reply:
Just checking to see if opt.InputFiles is empty and then fail is enough. 

It's just odd having it check the first element of the array.

Reply:
It's now ready